Bluetooth/Bluetooth.cpp

```diff
@@ -153,6 +156,8 @@ namespace WPEFramework
             registerMethod(METHOD_SET_DISCOVERABLE, &Bluetooth::setDiscoverableWrapper, this);
             registerMethod(METHOD_GET_NAME, &Bluetooth::getNameWrapper, this);
             registerMethod(METHOD_SET_NAME, &Bluetooth::setNameWrapper, this);
+            registerMethod(METHOD_SET_PROPERTIES, &Bluetooth::setPropertiesWrapper, this);
```

Comment:
Pls add them to version 2 of the plugin since these are new APIs. You can refer to how to add new versions here - https://github.com/rdkcentral/rdkservices/blob/sprint/2010/SystemServices/SystemServices.cpp#L361